app/workers/publishing_api_worker.rb

```diff
@@ -39,4 +44,13 @@ def save_draft_of_unpublished_edition(unpublishing)
       Whitehall::PublishingApi.save_draft_async(draft)
     end
   end
+
+  def handle_error(error)
+    if error.code >= 500
+      raise error
+    else
+      explanation = "The message is a duplicate and does not need to be retried"
+      Airbrake.notify_or_ignore(error, parameters: { explanation: explanation })
```

Comment:
@boffbowsh: This is based on publishing-api work here: https://github.com/alphagov/publishing-api/commit/3dd1b6bae628e32de477fc797e4c0666dad2827e we've used to successfully reduce the noise coming from doomed retries.
The documented example I found for [`notify_or_ignore`](https://github.com/airbrake/airbrake/wiki/Using-Airbrake-with-plain-Ruby) states _You can also pass a hash to Airbrake.notify_or_ignore method and store whatever you want_ this differs from the [`notify`](http://www.rubydoc.info/github/thoughtbot/airbrake/Airbrake.notify) method you refer to.